Bricolage Grotesque: Version 1.001;gftools[0.9.32] added

emmamarichal opened the pull request:
c20ca32: [gftools-packager] Bricolage Grotesque: Version 1.001;gftools[0.9.32] added

* Bricolage Grotesque Version 1.001;gftools[0.9.32] taken from the upstream repo https://github.com/ateliertriay/bricolage at commit https://github.com/ateliertriay/bricolage/commit/c3739c1d4ee94e1b3c130453b74a52a1cb5062b0.

RosaWagner commented:
@emmamarichal The font name is a bit long: `Bricolage Grotesque 96pt Condensed ExtraBold`. `12pt ExtraLight` could be a better fit within the available origins, no?

chrissimpkins commented on ofl/bricolagegrotesque/METADATA.pb:
```diff
@@ -34,5 +34,5 @@ axes {
   repository_url: "https://github.com/ateliertriay/bricolage"
-  commit: "95ddb0da9087a5f49f4616faea519e9e27d80a99"
+  commit: "c3739c1d4ee94e1b3c130453b74a52a1cb5062b0"
 }
```
Can add `minisite_url: https://ateliertriay.github.io/bricolage/` to the metadata file.  Mathieu confirmed that this will be the prod URL for the minisite

emmamarichal commented:
@RosaWagner Ah yes indeed, I'll update it, thanks! 
@chrissimpkins if Mathieu merge my new PR quickly, this one can be merge before the weekend! I'll update the metadata too

emmamarichal commented:
Updated

## Bricolage Grotesque: Version 1.001;gftools[0.9.33.dev8+g029e19f] added

---
 b9f6c71: [gftools-packager] Bricolage Grotesque: Version 1.001;gftools[0.9.33.dev8+g029e19f] added

* Bricolage Grotesque Version 1.001;gftools[0.9.33.dev8+g029e19f] taken from the upstream repo https://github.com/ateliertriay/bricolage at commit https://github.com/ateliertriay/bricolage/commit/84745e5b96261ae5f8c6c856e262fe78d1d6efdd.

 7b61b05: Added minisite url in metadata


 c300561: corrected metadata


 bb42ec1: corrected metadata

emmamarichal commented:
@RosaWagner @chrissimpkins, font origin updated (to make the name shorter), and metadata also!
